File: Motley/motley/solutions/security_stack.py
from aws_cdk import (
    # Duration,
    Stack, RemovalPolicy, )
from constructs import Construct

from motley.components.security.secrets_stack import SecretsStack


class SecurityStack(Stack):

    def __init__(self, scope: Construct, construct_id: str,
                 removal_policy: RemovalPolicy = RemovalPolicy.RETAIN,
                 **kwargs) -> None:
        super().__init__(scope, construct_id, **kwargs)

        self.secret = SecretsStack(self, "SecretsStack", removal_policy=removal_policy).secret

File: TypeScript/lib/stacks/security-stack.ts
import * as cdk from 'aws-cdk-lib';
import { Construct } from 'constructs';
import { IamNestedStack } from '../components/security/iam-nestedstack';



export class SecurityStack extends cdk.Stack {

    constructor(scope: Construct, id: string, props?: cdk.StackProps) {
        super(scope, id, props);

        let IamStack = new IamNestedStack(this, 'IamNestedStack', {
            RemovalPolicy: cdk.RemovalPolicy.DESTROY,
        })
    }
}
